receiver/githubreceiver/config_test.go

```diff
@@ -40,6 +43,14 @@ func TestLoadConfig(t *testing.T) {
 	defaultConfigGitHubScraper.(*Config).Scrapers = map[string]internal.Config{
 		githubscraper.TypeStr: (&githubscraper.Factory{}).CreateDefaultConfig(),
 	}
+	defaultConfigGitHubScraper.(*Config).WebhookConfig = webhook.Config{
```

Comment:
Webhook should be separate from `defaultConfigGitHubScraper`.

Reply:
I renamed this `defaultConfigGithubReceiver to make it more clear that it is the top-level configuration struct. I also renamed `WebhookConfig` to `Tracing`.